buildflow/core/app/runtime/actors/pipeline_pattern/pipeline_pool.py

```diff
@@ -124,11 +126,23 @@ async def snapshot(self) -> PipelineProcessorSnapshot:
         replica_snapshots: List[PullProcessPushSnapshot] = []
         # TODO: Dont access self.replicas directly. It should be accessed via a method
         # interface
-        for replica in list(self.replicas.values()):
-            snapshot: PullProcessPushSnapshot = (
-                await replica.ray_actor_handle.snapshot.remote()
-            )
-            replica_snapshots.append(snapshot)
+        dead_replica_idxs = []
```

Comment:
nit: idxs -> ids?

Reply:
supposed to be short for indices I'll just change it to that